pkgs/kde/gear/kdenlive/default.nix

```diff
@@ -31,7 +34,12 @@ mkKdeDerivation {
       url = "https://invent.kde.org/multimedia/kdenlive/-/commit/8be0e826471332bb739344ebe1859298c46e9e0f.patch";
       hash = "sha256-5hLePH5NlO4Lx8lg9kjBPI4jTmP666RGplaVCmS/9TA=";
     })
-  ];
+  ] ++ lib.optional enableGlaxnimate (
+    substituteAll {
+      src = ./dependency-paths-glaxnimate.patch;
```

Comment:
Because this patch is separate (again, why?)

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/324552